lib/render/VolumeOSPRay.cpp

```diff
@@ -777,7 +777,7 @@ OSPVolume VolumeOSPRay::_loadVolumeUnstructured(const Grid *grid)
     long added[32] = {0};
     long skipped[32] = {0};
 
-    //    int maxCells = std::min((int)nCells, (int)GetActiveParams()->GetValueLong("osp_max_cells", 1));
+    //    int maxCells = std::max(5, std::min((int)nCells, (int)GetParams()->GetValueLong("osp_max_cells", 1)));
```

Comment:
can this line of comment be removed?

Reply:
I ifdeffed it out along with the other debug code since it will be useful in the future.